app/controllers/episodes_controller.rb

```diff
@@ -1,11 +1,48 @@
 class EpisodesController < ApplicationController
   load_and_authorize_resource
+  before_action :set_episode, only: %i[edit show]
 
   def index
     @episodes = Episode.order(created_at: :desc).page params[:page]
   end
 
-  def show
+  def show; end
+
+  def edit; end
+
+  def create
+    @episode = Episode.new(episode_params.merge(created_by: current_user))
+
+    if @episode.save
+      redirect_to episodes_url, notice: 'Successfully created'
+    else
+      render :new, notice: 'Something wrong'
+    end
+  end
+
+  def update
+    if Episode.find(params[:id]).update(episode_params)
+      redirect_to episode_path
+    else
+      redirect_to edit_episode_path
+    end
+  end
+
+  private
+
+  def set_episode
     @episode = Episode.find(params[:id])
   end
+
+  def episode_params
+    params[:video] = params[:video].split('=').last
+    params[:date] = datetime
+    params.delete(:time)
```

Comment:
You don't need this delete

Reply:
Why? I don't have field `time` in Episode model.